docker/docker-compose.prod.yaml

```diff
@@ -11,6 +11,15 @@ services:
 
   celeryworker:
     restart: unless-stopped
+    # Set lower CPU priority to prevent blocking web service
```

Comment:
Can you add a comment to explain how one could temporarily disable this feature (for example, by commenting out a few lines within)?  Say we need the celery worker to populate a cache on deploy and want to run in `mean` mode?

Reply:
after conversation, i don't see this as needed.  nice work @ivan-c.